src/scaffold.ts

```diff
@@ -408,14 +409,32 @@ async function pickTemplate(
   return pickedItem?.value;
 }
 
-export async function getTemplatesList(collection?: string): Promise<ScaffoldV1TemplateList> {
+export async function getTemplatesList(
+  sanitizeOptions?: boolean,
+  collection?: string,
+): Promise<ScaffoldV1TemplateList> {
```

Comment:
More straightforward param ordering since `collection` affects the primary template listing, whereas the sanitizing is a secondary option affecting the templates listed:
```suggestion
export async function getTemplatesList(
  collection?: string,
  sanitizeOptions: boolean = false,
): Promise<ScaffoldV1TemplateList> {
```